pkgs/development/ocaml-modules/hxd/default.nix

```diff
@@ -1,36 +1,37 @@
 { lib, buildDunePackage, fetchurl
-, dune-configurator, cmdliner, angstrom
-, rresult, stdlib-shims, fmt, fpath
+, dune-configurator, cmdliner
+, withLwt ? true, lwt
```

Comment:
```suggestion
, ocaml, withLwt ? lib.versionAtLeast ocaml.version "4.07", lwt
```

Reply:
Out of curiosity: What is the issue with 4.06 and lwt?

Reply:
Syntax error.

Reply:
Thanks